src/main/scala/com/linagora/openpaas/gatling/unifiedinbox/scenari/SendEmailScenari.scala

```diff
@@ -20,11 +20,11 @@ object SendEmailScenari {
       .exec(registerSocketNamespaces)
       .exec(openWsConnection())
       .exec(loadOpeningEventTemplates)
-      .exec(generateJwtToken)
-      .exec(generateJwtToken)
-      .exec(generateJwtToken)
-      .exec(generateJwtToken)
-      .exec(generateJwtToken)
+      .exec(generateJwtTokenWithAuth)
+      .exec(generateJwtTokenWithAuth)
+      .exec(generateJwtTokenWithAuth)
+      .exec(generateJwtTokenWithAuth)
+      .exec(generateJwtTokenWithAuth)
```

Comment:
Why getting the JWT token 5 times?

Why getting mailboxes twice below?

Reply:
It was a capture of what openpaas was doing.
We should update it, maybe the thing have improved a little bit.
But it is not in the scope of this ticket